ru/browser/browser/zenAppExtensionFields.ftl

```diff
@@ -1,6 +1,6 @@
 
 extension-firefox-compact-galaxy-name = Zen Galaxy
-extension-firefox-compact-galaxy-description = Тема с темной цветовой схемой и приятным прикосновением галактики!
+extension-firefox-compact-galaxy-description = Тема в темной цветовой гамме с нотками межгалактической неизвданности!
```

Comment:
Just a small typo from me... 😅 

- [Неизведанность on Wiktionary](https://ru.wiktionary.org/wiki/%D0%BD%D0%B5%D0%B8%D0%B7%D0%B2%D0%B5%D0%B4%D0%B0%D0%BD%D0%BD%D0%BE%D1%81%D1%82%D1%8C) just in case

```suggestion
extension-firefox-compact-galaxy-description = Тема в темной цветовой гамме с нотками межгалактической неизведанности!
```